src/pages/submit/Submit/Submit.tsx

```diff
@@ -0,0 +1,29 @@
+import { Complete } from '@assets/images';
+import { ConfirmButton } from '@pages/submit';
+import { useLocation, useNavigate } from 'react-router-dom';
+import styles from './Submit.module.css';
+
+const Submit: React.FC = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const { firstCardNumber, cardIssuer } = location.state;
+
+  return (
+    <div className={styles.submitContainer}>
+      <img className={styles.complete} src={Complete} alt="Complete" />
+      <p className={styles.message}>
+        <span>{`${firstCardNumber}로 시작하는`}</span>
+        <br />
+        {`${cardIssuer}가 등록되었어요.`}
```

Comment:
`cardIssuer`에 카드에 매칭되는 숫자가 들어있어서 화면이 다음과 같이 나오네요 

<img width="417" alt="image" src="https://github.com/woowacourse/react-payments/assets/44080404/b0eee635-b4ed-47ca-9627-087c8fae8699">

Reply:
감사합니다! 다음과 같이 해결하였습니다!

```suggestion
       {`${CARD.ISSUER_NAME[cardIssuer]}가 등록되었어요.`}
```